osu.Desktop/Program.cs

```diff
@@ -31,19 +30,9 @@ public static class Program
         [STAThread]
         public static void Main(string[] args)
         {
-            /*
```

Comment:
The Velopack docs suggest that you [*must*](https://arc.net/l/quote/urfqwedh) run it before everything else. However, as you found, that's not necessarily a strict requirement. In another example on Velopack's docs, code is [executed](https://arc.net/l/quote/chgyexjh) before the `.Run()`

As for me, I decided "So you should put `setupVelopack()` before everything else, but putting stuff before it isn't the end of the world as it would've been with Squirrel. So I'll tone it down a bit, and just keep the advisory."

Though, @caesay would likely know best about this. I'd wait for his thoughts.

Reply:
I've made the inline comment a bit louder for now. Probably fine to continue.